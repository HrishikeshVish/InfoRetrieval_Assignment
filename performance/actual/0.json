["https://archive.org/details/MSNBCW_20191215_150000_AM_Joy#start/6848/end/6883", "https://archive.org/details/MSNBCW_20190811_020000_All_In_With_Chris_Hayes#start/3440/end/3475", "https://archive.org/details/CNN_20091210_010000_Campbell_Brown#start/2153/end/2188", "https://archive.org/details/MSNBCW_20190923_100000_Morning_Joe#start/6997/end/7032", "https://archive.org/details/CNN_20100502_020000_CNN_Newsroom#start/2017/end/2052", "https://archive.org/details/MSNBCW_20191229_120000_Weekends_With_Alex_Witt#start/2735/end/2770", "https://archive.org/details/MSNBCW_20181211_110000_Morning_Joe#start/1270/end/1305", "https://archive.org/details/MSNBCW_20191224_210000_Deadline_White_House#start/803/end/838", "https://archive.org/details/MSNBCW_20170805_010000_The_Rachel_Maddow_Show#start/2579/end/2614", "https://archive.org/details/FOXNEWSW_20130109_230000_Special_Report_With_Bret_Baier#start/32/end/67"]