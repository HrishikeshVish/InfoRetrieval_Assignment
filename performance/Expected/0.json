["https://archive.org/details/CNN_20100502_020000_CNN_Newsroom#start/2017/end/2052", "https://archive.org/details/MSNBCW_20181211_110000_Morning_Joe#start/1270/end/1305", "https://archive.org/details/MSNBCW_20190811_020000_All_In_With_Chris_Hayes#start/3440/end/3475", "https://archive.org/details/CNN_20091210_010000_Campbell_Brown#start/2153/end/2188", "https://archive.org/details/MSNBCW_20170805_010000_The_Rachel_Maddow_Show#start/2579/end/2614", "https://archive.org/details/FOXNEWSW_20130109_230000_Special_Report_With_Bret_Baier#start/32/end/67", "https://archive.org/details/CNNW_20170723_140000_Fareed_Zakaria_GPS#start/1095/end/1130", "https://archive.org/details/MSNBCW_20141112_230000_PoliticsNation#start/488/end/523", "https://archive.org/details/MSNBCW_20190923_100000_Morning_Joe#start/6997/end/7032", "https://archive.org/details/FOXNEWS_20091130_140000_Americas_Newsroom#start/5710/end/5745"]